<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sensor_act">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>s1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="schuifdeurKnop">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>vd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deur_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>d1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deur_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>d2</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="kaartenbak">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>20</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>68</green>
          <blue>68</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditS">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>280</y>
      <width>113</width>
      <height>34</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>68</green>
          <blue>68</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditD1">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>320</y>
      <width>113</width>
      <height>34</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>68</green>
          <blue>68</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditD2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>360</y>
      <width>113</width>
      <height>34</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>68</green>
          <blue>68</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditA">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>180</y>
      <width>113</width>
      <height>34</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>68</green>
          <blue>68</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>40</red>
          <green>40</green>
          <blue>40</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QPushButton" name="AP">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AN">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ToonKaart">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>118</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Toon Kaart</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGebouw">
    <property name="title">
     <string>Gebouw</string>
    </property>
    <addaction name="actionGebouw"/>
    <addaction name="actionSlot"/>
    <addaction name="actionKaartenbalk"/>
   </widget>
   <addaction name="menuGebouw"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGebouw">
   <property name="text">
    <string>Gebouw</string>
   </property>
  </action>
  <action name="actionSlot">
   <property name="icon">
    <iconset theme="system-lock-screen">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Slot</string>
   </property>
  </action>
  <action name="actionKaartenbalk">
   <property name="text">
    <string>Kaartenbalk</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
